src/mrpro/operators/FourierOp.py

```diff
@@ -223,3 +224,23 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor,]:
             x = x.permute(*unpermute)
 
         return (x,)
+
+    def __repr__(self):
+        """Representation method for Fourier Operator."""
+        if self._nufft_dims:
+            string = (
+                f'Dimension along which NUFFT is applied: {self._nufft_dims}\n'
+                f'Dimension which is ignored in NUFFT: {self._ignore_dims!s}\n'
+            )
+        else:
+            string = ''
+        if self._fft_dims:
+            string += (
+                f'{self._fast_fourier_op}\n'
+                f'Dimension which is ignored in FFT: {self._ignore_dims!s}\n'
+                f'{self._cart_sampling_op}\n'
+            )
+        device = str(self._traj.device)
```

Comment:
should the device not be availabe from the operator rather than the trajectory? the operator can be put to cuda after creating from cpu-traj, can't it?